tests/views/test_sign_in.py

```diff
@@ -149,4 +150,16 @@ def test_sign_out_deleting_session_variables(self):
         self.assertEqual(response.status_code, 200)
         self.assertNotIn(b"Sign out", response.data)
         with self.client.session_transaction() as session:
-            self.assertEqual(session.get("next"), None)
+            self.assertIsNone(session.get("next"))
+
+        @requests_mock.mock()
+        def test_session_holds_permission_expiry_as_iso8601_date(self, mock_request):
```

Comment:
This is a detached test of datetime.fromisoformat and not testing our code. It should realistically be hitting line 472 on uaa_controller.py, it's just the code used on that line is the same as this at the moment. If in the future that line was changed, this test is affectively redundant or is not right